cypress/features/regression/designSystem/menu.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Design Systems Menu component
+  I want to test the Design Systems Menu component
+    
+  @positive
+  Scenario Outline: Scenario name: Test that all child link elements are part of a scrollable block
+      Given I open "Design System Menu" component page "scrollable-submenu" in no iframe
+      When I open <submenu number>
+      Then The submenu is scrollable
```

Comment:
We just scroll here, but we actually need to be sure that submenu was scrolled to the proper element. So we need to add something that some element became visible after scroll.